node/config/load.go

```diff
@@ -46,7 +70,93 @@ func FromReader(reader io.Reader, def interface{}) (interface{}, error) {
 	return cfg, nil
 }
 
-func ConfigUpdate(cfgCur, cfgDef interface{}, comment bool) ([]byte, error) {
+// function from raw config string to loadability
+type loadabilityCheck struct {
```

Comment:
(can we kill the term "loadability check")

Reply:
Basically, we're doing two things here:

1. We're validating the config.
2. We're providing a default config, if applicable.

I'm assuming that "loadability check" is "can we load this config", but it's just really confusing.